perftracker/models/regression.py

```diff
@@ -38,11 +38,14 @@ def __str__(self):
     def save(self):
         super(RegressionModel, self).save()
 
-    def pt_get_jobs(self):
-        return JobModel.objects.filter(regression_tag=self.tag, project=self.project, deleted=False).order_by('end').all()
+    def pt_get_all_jobs(self):
+        return JobModel.objects.filter(regression_original=self, deleted=False).order_by('end').all()
+
+    def pt_get_linked_jobs(self):
+        return JobModel.objects.filter(regression_linked=self, deleted=False).order_by('end').all()
 
     def pt_set_first_last_job(self):
-        jobs = self.pt_get_jobs()
+        jobs = self.pt_get_all_jobs()
```

Comment:
Seems like a bug. If you exclude first job in the regression, the regression "start date" will still point to the date of the excluded job. and the first job will be still the "excluded" one.

The whole idea of job exclusion is to "remove" it completely from the regression, like the job doesn't exist. 
But we sill need an ability to return the job back if it was excluded accidentally, that is why we need pt_get_all_jobs()